src/TestSuite/TestCase.php

```diff
@@ -898,6 +898,24 @@ public static function setAppNamespace(string $appNamespace = 'TestApp'): ?strin
         return $previous;
     }
 
+    /**
+     * Adds a fixture to this test case.
+     *
+     * Examples:
+     * - core.Tags
+     * - app.MyRecords
+     * - plugin.MyPluginName.MyModelName
+     *
+     * @param string $fixture Fixture
+     * @return $this
+     */
+    public function addFixture(string $fixture)
```

Comment:
Should this be protected? Any reason to have this public?

Reply:
Can't think of any reason for it to be public.

Reply:
Fixed.